### PR TITLE
Lesson hw 3

### PR DESCRIPTION
**Pull request by kireeva67**

### CODE REVIEW

File: hw/hw3/task3.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+class Users {
+  constructor(numberOfUsers) {
+    this.numberOfUsers = numberOfUsers;
+  }
+
+  async getUsers() {
+    let url = `https://randomuser.me/api/?results=${this.numberOfUsers}`;
+    let usersData = await this.makeGetRequest(url);
+    let users = "";
+    for (let key in usersData["results"]) {
```
Comment (alexmaltsv) [inline hw/hw3/task3.js]: Better to use reduce instead of `for in`

File: hw/hw3/task1.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+"use strict";
+class StringFormatter {
+  removeNonUniqueChars(string) {
+    let stringWithoutDoubleChars = "";
+    for (let counter = 0; counter < string.length; counter++) {
```
Comment (alexmaltsv) [inline hw/hw3/task1.js]: better to use reduce here

File: hw/hw3/task1.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  }
+}
+
+const stringWithoutDoubleChars = new StringFormatter();
+console.log(stringWithoutDoubleChars.removeNonUniqueChars("Hello,, world55"));
+
+class StringFormatterForNumbers extends StringFormatter {
+  removeNonUniqueChars(string) {
+    let stringWithoutDoubleNumbers = "";
+    for (let counter = 0; counter < string.length; counter++) {
```
Comment (alexmaltsv) [inline hw/hw3/task1.js]: the same

**Review by alexmaltsv — APPROVED** [review]

File: hw/hw3/task3.js (hunk 1 above)
Comment (kireeva67) [reply hw/hw3/task3.js]: fixed

File: hw/hw3/task1.js (hunk 2 above)
Comment (kireeva67) [reply hw/hw3/task1.js]: fixed

File: hw/hw3/task1.js (hunk 3 above)
Comment (kireeva67) [reply hw/hw3/task1.js]: fixed

**Pull request merged**